src/genre-select/index.jsx

```diff
@@ -0,0 +1,42 @@
+import './style.css'
+
+import {useState} from 'react'
+
+const GenreSelect = () => {

```

Comment:
according to the task AC, this component should have 2 properties. here there is none